active-directory-wpf-msgraph-v2/MainWindow.xaml.cs

```diff
@@ -49,8 +50,9 @@ private async void CallGraphButton_Click(object sender, RoutedEventArgs e)
 
                 try
                 {
-                    authResult = await app.AcquireTokenInteractive(scopes, this)
+                    authResult = await app.AcquireTokenInteractive(scopes)
                         .WithAccount(accounts.FirstOrDefault())
+                        .WithParentActivityOrWindow(new WindowInteropHelper(this).Handle) // optional, used to center the browser on the window
```

Comment:
Can't we just pass this?

Reply:
Yes, MSAL already takes a dependency on WinForms (and btw, website scenario customers are suprised that it does). We don't want to take a depnency on System.Windows (WPF) too.

Reply:
Btw I tested with the handle and the centering algorithm works well.

Reply:
If we want to the sample to be simpler, we can just remove it.

Reply:
No, that's great.
We just need to update the wiki